enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,430 @@
+---
+title: installer-telemetry-metrics
+authors:
+  - "@patrickdillon"
+  - "@rna-afk"
+reviewers:
+  - "@abhinavdahiya"
+  - "@brancz"
+  - "@crawford"
+  - "@sdodson"
+approvers:
+  - "@abhinavdahiya"
+  - "@brancz"
+  - "@crawford"
+  - "@sdodson"
+creation-date: 2020-02-07
+last-updated: 2020-02-07
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Installer Telemetry Metrics
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+ > 1. How to filter to collect only real user usage and prevent OpenShift developer/Red Hat internal users from skewing metrics? Perhaps filter by version?
+ > 2. Is it possible to determine whether a user modified an ignition config or manifest asset by comparing the asset the installer would generate to the one found on disk?
```

Comment:
nit: "ignition" -> "Ignition".

And yeah, you can see which assets are dirty [here][1], although we don't currently cache that information on the `Asset` itself for future reference.

[1]: https://github.com/openshift/installer/blob/b518ee89d4ee64f95b3b9401b25439ce65480189/pkg/asset/store/store.go#L310-L315